libslide/src/bignum.rs

```diff
@@ -60,8 +67,12 @@ impl fmt::Display for Bignum {
         if self.is_neg {
             result.push('-');
         }
-        for n in self.int.iter().rev() {
-            result.push(to_char(*n));
+        if self.int.is_empty() {
+            result.push('0');
```

Comment:
Why?

Reply:
I want decimals to be printed 0.1 instead of .1

Reply:
+1